### PR TITLE
Add the LSTM model

### PR DESCRIPTION
**Pull request by 1DWalker**

### CODE REVIEW

**Comment by Expertium** [comment]
I thought you were going to use more input features, such as the time of the day and the number of reviews done on that day before the current review (this stuff: https://imgur.com/a/Qrl5tEV).

Anyway, I think it's time we add a column for input features.

**Comment by 1DWalker** [comment]
I mentioned that I tried those features but they increased train accuracy but decreased validation accuracy. It just helped the model to overfit to each user a little. If I try that again then I would need more regularization or I would train on more users first. I've left some of the code that creates those features in the `create_features_helper` function in case anyone wants to use it in the future.

@L-M-Sherlock thoughts on additional columns in the README tables to indicate which features are used by which model? It is related to https://github.com/open-spaced-repetition/srs-benchmark/issues/161.

**Comment by L-M-Sherlock** [comment]
The README is too long now. I recommend recording it in wiki.

**Comment by L-M-Sherlock** [comment]
Do you mind merging it at first?

**Comment by 1DWalker** [comment]
I'm not sure what you mean by that. There should be no conflicts with the main branch I think.

**Comment by L-M-Sherlock** [comment]
I mean we can improve the document in the next PR or build the wiki. This PR looks good to me now.
